pom.xml

```diff
@@ -34,8 +34,8 @@
     <jenkins.version>2.402</jenkins.version>
     <selenium.version>4.9.0</selenium.version>
     <guava.version>31.1-jre</guava.version> <!-- aligned with selenium -->
-    <aether.version>1.1.0</aether.version>
-    <maven.version>3.3.9</maven.version>
+    <maven.version>3.8.1</maven.version>
```

Comment:
Is there a specific reason to use 3.8.1 over 3.9.1?

Reply:
Yes. All of our Maven plugins depend on version 3.8.1 so I am sticking with a consistent baseline throughout the Jenkins ecosystem.